src/deluge/gui/ui_timer_manager.cpp

```diff
@@ -144,6 +153,16 @@ void UITimerManager::routine() {
 					setTimer(TIMER_GRAPHICS_ROUTINE, 15);
 					break;
 
+				case TIMER_AUTOMATION_VIEW: //timer to redisplay the parameter name on the screen in automation view
+					if (getCurrentUI() == &automationInstrumentClipView
+					    && (((InstrumentClip*)currentSong->currentClip)->lastSelectedParamID != 255)) {
```

Comment:
255 here should be `kNoLastSelectedParamId` probably.

Reply:
done